tests/snuba/tagstore/test_backend.py

```diff
@@ -80,51 +78,6 @@ def setUp(self):
 
         assert requests.post(snuba.SNUBA + '/tests/insert', data=data).status_code == 200
 
-    @responses.activate
```

Comment:
I guess it can, but I don't think we're going to mock any other endpoints. In general we switched to testing against "real" Snuba once I added the ability in Travis.